ssl/statem/extensions.c

```diff
@@ -1709,6 +2017,13 @@ int tls_psk_do_binder(SSL_CONNECTION *s, const EVP_MD *md,
     OPENSSL_cleanse(finishedkey, sizeof(finishedkey));
     EVP_PKEY_free(mackey);
     EVP_MD_CTX_free(mctx);
+#ifndef OPENSSL_NO_ECH
+    EVP_MD_CTX_free(ctx);
+    if (tpkt_mem != NULL) {
+        WPACKET_cleanup(&tpkt);
+        BUF_MEM_free(tpkt_mem);
+    }
```

Comment:
Is this a left over from an earlier iteration? We clean this stuff up, but didn't use it anywhere.

Reply:
yep, looks like it - gone now